<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreatePatientPage</class>
 <widget class="QWidget" name="CreatePatientPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string>Name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="name_lineEdit">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>22</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>22</height>
              </size>
             </property>
             <property name="inputMask">
              <string/>
             </property>
             <property name="maxLength">
              <number>50</number>
             </property>
             <property name="frame">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string>contact</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="contact_lineEdit">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>22</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>22</height>
              </size>
             </property>
             <property name="inputMask">
              <string/>
             </property>
             <property name="maxLength">
              <number>50</number>
             </property>
             <property name="frame">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string>Gender</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="gender_ComboBox">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>22</height>
              </size>
             </property>
             <item>
              <property name="text">
               <string>M</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>F</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string>Birth Date</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="date_dateEdit">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>22</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>22</height>
              </size>
             </property>
             <property name="frame">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Medical History:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="medicalHistory_plainTextEdit">
           <property name="minimumSize">
            <size>
             <width>256</width>
             <height>192</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>256</width>
             <height>192</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="add_pushButton">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 173, 181);
font: 700 12pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
